tools/template_based_query_generation/src/main/java/FeatureType.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Types of queries in SQL language encoding
+ * feature_root, ddl_feature_root, dml_feature_root, dql_feature_root, feature_sink are types to help create reference nodes
+ */
+public enum FeatureType {
+	feature_root, // type for the root node
```

Comment:
Sorry for the confusion! Let's still keep those types upper case

Reply:
got it!